pkg/pillar/persistcache/persistcache_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package persistcache_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/lf-edge/eve/pkg/pillar/persistcache"
+	"gotest.tools/assert"
```

Comment:
Is there something missing in the test package we use in all the other unit tests?

Reply:
No, I removed assert completely and used gomega